src/main/java/com/bringframework/exceptions/NoUniqueBeanException.java

```diff
@@ -0,0 +1,8 @@
+package com.bringframework.exceptions;
+
+/**
+ * Exception thrown when multiple beans of the same type are found but expected one.
+ */
+public class NoUniqueBeanException extends RuntimeException {
```

Comment:
I suggest to rename  to `UniqueBeanException ` or `MultipleBeanException`

Reply:
1. As for me, the NoUniqueBeanException name fully covers the problems that may arise and does not give a field for fantasies regarding other problems that will not be related to this error 
2. UniqueBeanException - if without context and docs, then the first in the head there are problems that are directly opposite to what should be. 
3. MultipleBeanException also gives a large field of problems, most unrelated to our mistake. 
4. As an argument, the name NoUniqueBeanException is proposed by Taras.
5. And even in spring, the concept of this exception is at the level of creating beans from BeanDefinitions and there it is called NoUniqueBeanDefinitionException

Reply:
Okay, I am fine with it